tpr/tpr.go

```diff
@@ -14,6 +14,10 @@ import (
 
 const (
 	tprInitMaxElapsedTime = 2 * time.Minute
+
+	// resyncPeriod is the interval at which the operator cache is invalidated,
+	// and the lister function is called.
+	resyncPeriod = 1 * time.Minute
```

Comment:
I think it should be capitalized :)

BTW compilation should fail I think and CI passes...